test/suites/storage/snapshot.robot

```diff
@@ -78,6 +80,7 @@ Test Case Teardown
     Oc Delete    pvc snapshot-restore -n ${NAMESPACE}
     Named PVC Should Be Deleted    test-claim-thin
     Named PVC Should Be Deleted    snapshot-restore
+    Oc Delete    ns ${NAMESPACE}
```

Comment:
Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82